### ISSUE TITLE
Cluster kubelet version

### ISSUE THREAD

**Issue by etenzy** [syseleven/terraform-provider-metakube#30]
My Testcluster:  1.18.6

tf plan (without changes)
```
│ Error: 1 error occurred:
│       * unknown version for node deployment 1.18.16, available versions [1.17.17 1.18.18]
│  
│  
│  
│   with metakube_node_deployment.deployment, 
│   on main.tf line 55, in resource "metakube_node_deployment" "deployment": 
│   55: resource "metakube_node_deployment" "deployment" {
```

changed the k8s_version to 1.18.18
```
│ Error: 1 error occurred:
│       * node deployment version (1.18.18) cannot be greater than cluster version (1.18.16)
│ 
│ 
│ 
│   with metakube_node_deployment.deployment,
│   on main.tf line 55, in resource "metakube_node_deployment" "deployment":
│   55: resource "metakube_node_deployment" "deployment" {
```

```bash
tf plan -target metacube_cluster.cluster
-> No changes. Infrastructure is up-to-date.
```
This leads me to suspect that an upgrade of the "Master Version" via terraform is not possible.

After changing the "Master Version" in the Metakube-UI everything works fine again.

In my opinion the proper practice would be to keep older versions available, or making the "Master Version" upgradable.

**Comment by furkhat** [comment]
Thanks for reporting! Yes I think validation should be improved by allowing to have node deployment with older kubelet version if it is already exist.

**Comment by furkhat** [comment]
Fixed in `0.2.6` release.
